src/style.css

```diff
@@ -1,3 +1,132 @@
-body {
+body,
+h2,
+p {
   margin: 0;
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+.main {
+  display: flex;
+  justify-content: center;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  align-items: start;
+}
+
+.card {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+}
+
+.card__photo {
+  width: 160px;
+  height: 134px;
+  padding-top: 33px;
+}
+
+.card__about {
+  display: flex;
+  padding: 40px 0 16px;
+  justify-content: center;
+  flex-direction: column;
+}
+
+.card__title {
+  line-height: 18px;
+  color: #060b35;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.card__code {
+  color: #616070;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  padding-top: 4px;
+}
+
+.stars {
+  display: flex;
+  align-items: center;
+  width: 166px;
+  justify-content: space-between;
+  padding-top: 16px;
+}
+
+.stars__rating {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding-left: 2px;
+  padding-top: 2px;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+  margin-right: 4px;
+
+}
+
+.stars__star:nth-child(1n+5) {
+  background-image: url(./images/star.svg);
+}
```

Comment:
reuse stars styles from a `stars` task. So your solution should be flexible with different amount of active stars

Reply:
Still not fixed from the previous review